### PR TITLE
libtommath, libtomcrypt: new website for libtom project

### PR DESCRIPTION
**Pull request by apjanke**
STATUS: Don't merge yet; status of upstream sites is unclear.

Per their website, now at www.libtom.net:

> This website is back, and libtom has found new maintainers.

### CODE REVIEW

File: Library/Formula/libtomcrypt.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class Libtomcrypt < Formula
   desc "Modular and portable cryptographic toolkit"
-  # homepage/url down since ~May 2015
-  homepage "http://libtom.org/?page=features&whatfile=crypt"
-  url "http://libtom.org/files/crypt-1.17.tar.bz2"
+  homepage "http://www.libtom.net/?page=features&newsitems=5&whatfile=crypt"
+  url "http://www.libtom.net/files/crypt-1.17.tar.bz2"
   mirror "https://distfiles.macports.org/libtomcrypt/crypt-1.17.tar.bz2"
   sha256 "e33b47d77a495091c8703175a25c8228aff043140b2554c08a3c3cd71f79d116"
+  head "https://github.com/libtom/libtomcrypt"
```
Comment (DomT4) [inline Library/Formula/libtomcrypt.rb]: Need a `.git` on the end of this URL.

File: Library/Formula/libtommath.rb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 class Libtommath < Formula
   desc "C library for number theoretic multiple-precision integers"
-  # homepage/url down since ~May 2015
-  homepage "http://libtom.org/?page=features&newsitems=5&whatfile=ltm"
-  url "http://libtom.org/files/ltm-0.42.0.tar.bz2"
+  homepage "http://www.libtom.net/?page=features&newsitems=5&whatfile=ltm"
+  url "http://www.libtom.net/files/ltm-0.42.0.tar.bz2"
   mirror "https://distfiles.macports.org/libtommath/ltm-0.42.0.tar.bz2"
   sha256 "7b5c258304c34ac5901cfddb9f809b9b3b8ac7d04f700cf006ac766a923eb217"
+  head "https://github.com/libtom/libtommath"
```
Comment (DomT4) [inline Library/Formula/libtommath.rb]: Need a `.git` on the end of this URL.

**Comment by apjanke** [comment]
Repo URLs fixed.

`brew install --HEAD libtomcrypt` from this new URL is failing for me on 10.11, but it looks like the same error as the latest actual release in #43906.

**Comment by apjanke** [comment]
Bump... can this be pulled in? Previous issues are fixed and checks pass.

**Comment by dunn** [comment]
😰 http://bot.brew.sh/job/Homebrew%20Pull%20Requests/34227/

**Comment by apjanke** [comment]
D'oh! I'll check that out.

**Comment by apjanke** [comment]
Known issue: #43906; it's failing on 10.10 and 10.11.

Can we just merge this website fix since it's not a regression, and deal with the build failure under the other issue?

**Comment by sjaeckel** [comment]
Not a good idea to update that, as the owner of libtom.net is currently not reachable and I don't know when this will change.
I hope to get libtom.org back up running.

**Comment by apjanke** [comment]
How about now? It looks like libtom.net has the same contents as libtom.org now, and libtom.org is back online. Should we update this with links in to the new libtom.org site layout?

**Comment by sjaeckel** [comment]
Yes, libtom.net and libtom.org are functional now.
As I've only control over libtom.net you should probably go for that domain.
Download URLs are currently only from github.

**Comment by sjaeckel** [comment]
Regarding the site layout, yes that will be the one you want to update your links to.
You may ask youself why libtom.net is back - long story short, I was able to transfer it to me

**Comment by apjanke** [comment]
Cool. I've revised and rebased this PR to use libtom.net and reflect the new site structure and GitHub releases. Works for me locally. Look good?

Download URLs from GitHub are fine for us, as long as they're for tagged releases, like you're doing here.

The new `libtom.net` web site is still referencing `libtom.org`, both in the site text (e.g. "Welcome to the LibTom Projects home here at libtom.org" on the front page) and in link URLs (e.g. in the navigation bar). Is there a repo for the web site where we can put bug reports/PRs for this?

If these new URLs work out, I'll try to add a couple little tests as part of this PR so the formulae pass our CI.

**Comment by mikemcquaid** [comment]
@apjanke LGTM when :green_heart:

**Comment by DomT4** [comment]
Would be good to pull bottles here when you merge. The `libtomcrypt` is currently lacking an El Cap one.

**Comment by apjanke** [comment]
No bottles (at least for libtomcrypt), due to outstanding issue #43906, which breaks the build on 10.10 and 10.11.

I think the thing to do is just merge this anyway and deal with the build failure separately: this URL fix will at least let brew-using devs download the files and attempt the build so they can debug and fix it.

**Comment by DomT4** [comment]
Ah, alright. Feel free in that case!

**Pull request closed without merging**